.travis.yml

```diff
@@ -2,18 +2,14 @@ language: ruby
 rvm:
   - 2.2
   - 2.1
-  - 2.0
-matrix:
-  include:
-    - # Ruby 1.9
-      rvm: 1.9
-      env: JEKYLL_VERSION=2.0
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   matrix:
+    - JEKYLL_VERSION=3.3
     - JEKYLL_VERSION=3.0
     - JEKYLL_VERSION=2.5
+    - JEKYLL_VERSION=2.0
```

Comment:
I was trying to find what Jekyll versions were being supported. I guess now 3.x is up to 3.3, the 2.x branch is probably dead? Should be tested for legacy users I suppose... 

Does 2.5 uses the [2.5.3 release](https://github.com/jekyll/jekyll/releases/tag/v2.5.3)?